rules/sprint-review-retro-email/rule.md

```diff
@@ -81,6 +81,12 @@ Attendees: *(Optional as they may be in the to and CC)*
 
    ![Figure: Sprint Burndown](burndown.JPG)
 
+1. Timesheet data - Who worked this Sprint?
+
+:::img-medium
+![Figure: Timesheet data for this Sprint](https://github.com/SSWConsulting/SSW.Rules.Content/assets/38869720/1f1b821a-5e3a-4e3c-9b03-acbba09e00f2)
```

Comment:
HI @bradystroud 

1. Is this image in the right location?
2. Shouldn't it be added as part of this PR?